index.js

```diff
@@ -178,6 +178,10 @@ is.domElement = x => is.object(x) && x.nodeType === NODE_TYPE_ELEMENT && is.stri
 
 is.infinite = x => x === Infinity || x === -Infinity;
 
+const isAbsoluteMod2 = value => x => is.number(x) && Math.abs(x % 2) === value;
```

Comment:
`is.number()` => `is.integer()`, since per docs it only works on integers anyway.

Reply:
👍 Updated